components/process.rst

```diff
@@ -410,6 +410,46 @@ instead::
     );
     $process->run();
 
+Executing PHP child processes with the same configuration
+---------------------------------------------------------
+
+When you start a PHP process, it's started using its default `ini` settings. Let's assume you have a configured
+`memory_limit` of `256M` in your `php.ini` and you want to disable it when running your `bin/console` script to access
+the Symfony console without any memory limit. You can then dynamically override it using the`-d` command line option
+like so: `php -d memory_limit=-1 bin/console app:my-command`.
+
+Problem solved. However, let's assume you have an `app:my-command` that itself again, starts a PHP child process:
+
+    use Symfony\Component\Process\Process;
+
+    class MyCommand extends Command
+    {
+        protected function execute(InputInterface $input, OutputInterface $output): int
+        {
+            $childProcess = new Process(['bin/console', 'cache:pool:prune']);
+        }
```

Comment:
@alamirault open for a PR to remove this false positive too?

Reply:
I will 🙂